source/tests/utils_test.h

```diff
@@ -143,3 +140,15 @@ std::set<std::string> get_uris(const nt::Indexes& indexes, const nt::Data& data)
         const auto& r2 = range2; \
         BOOST_CHECK_EQUAL_COLLECTIONS(std::begin(r1), std::end(r1), std::begin(r2), std::end(r2)); \
     }
+
+
+namespace boost { namespace geometry { namespace model {
+inline std::ostream& operator<<(std::ostream& os, const navitia::type::Polygon& points){
+    os << "{\"type\":\"Polygon\",\"coordinates\":[[";
+    os << std::setprecision(16) << "[" << points.outer()[0].lon() << ", " << points.outer()[0].lat() << "],";
+    for(int i = 1; i <= 360; i++) {
```

Comment:
(just in case) hard coded range, is it what you want? How about `i < points.outer().size()`